### PR TITLE
Hotfix for docker image to support async logging

### PR DESCRIPTION
**Pull request by mdebarros**
- Added async logging support to DockerFile
- Updated dependencies for Jest
- Fixed vulnerability issues
- Bumped version to v10.1.1

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,5 +27,9 @@ USER ml-user
 COPY --chown=ml-user --from=builder /opt/quoting-service .
 RUN npm prune --production
 
+# Create empty log file & link stdout to the application log file
```
Comment (lewisdaly) [inline Dockerfile:30]: This _may_ need to be added before the `RUN adduser` step to make sure that the correct permissions are applied to the log file, but I'm not 100% sure.

Comment (mdebarros) [reply Dockerfile]: Yeah, I figured that out when testing it. I just pushed an update to resolve it.

I also tested this on dev2, it seems to be all good now.

File: Dockerfile [hunk 2]
```diff
@@ -20,12 +20,18 @@ FROM node:12.16.1-alpine
...
+RUN ln -sf /dev/stdout ./logs/combined.log
+
 # Create a non-root user: ml-user
 RUN adduser -D ml-user 
 USER ml-user
 
 COPY --chown=ml-user --from=builder /opt/quoting-service .
 RUN npm prune --production
 
+USER ml-user
```
Comment (lewisdaly) [inline Dockerfile:35]: I don't think we need this here

Comment (mdebarros) [reply Dockerfile]: Thanks, left some previous changes from testing.

**Pull request merged**